README.md

```diff
@@ -360,6 +370,17 @@ translation:
   api_key: <Google Translate API key>
 ```
 
+<a name="deepl-translation-config"></a>
+### DeepL Pro Translate
+
+`i18n-tasks translate-missing` requires a DeepL Pro API key, get it at [DeepL](https://www.deepl.com/pro).
+
+```yaml
+# config/i18n-tasks.yml
+translation:
+  api_key: <Deep Pro API key>
```

Comment:
Let's have separate settings for different API keys. `google_translate_api_key` and `deepl_api_key`. (`api_key` can then work as `google_translate_api_key` with a deprecation warning`)